benchmark/python/sparse/sparse_end2end.py

```diff
@@ -181,7 +220,7 @@ def row_sparse_pull(kv, key, data, slices, weight_array, priority):
     train_data = mx.io.LibSVMIter(data_libsvm=path, data_shape=(feature_dim,),
                                   batch_size=batch_size, num_parts=num_worker,
                                   part_index=rank)
-    if dummy_iter:
+    if dummy_iter or compute_only:
```

Comment:
Is DummyIter not used for communication_only????

Reply:
Nice catch. This was a mistake. Added communication_only.